tests/test_evaluator.py

```diff
@@ -36,22 +34,11 @@ def test_evaluator_valid_input() -> None:
         check=False,
     )
 
-    with open(output_path2, "w") as printed_output_file:
```

Comment:
What's the reasoning for getting rid of this part?

Reply:
This is provided in the commit message. i.e.

> We should not attempt a 1-to-1 match of stdout from running the code.
> This is not an API that we need to expect consistent output from (i.e.
> someone may change a print statement, this should not break tests).
> 
> Only test on the json output of the program.

Reply:
https://github.com/wizhaoredhat/ocp-traffic-flow-tests/pull/28#discussion_r1597217193